paddle/framework/operator.cc

```diff
@@ -99,5 +99,48 @@ std::string OperatorBase::DebugString() const {
   return ss.str();
 }
 
+template <>
+const Variable* OperatorContext::Input<Variable>(int index) const {
```

Comment:
Please use `size_t` instead of `int`. 

size_t is the standard type for stl container.